src/common/utils/FocusManager.ts

```diff
@@ -264,11 +280,17 @@ export abstract class FocusManager {
 
         Object.keys(this._myFocusableComponentIds).forEach(componentId => {
             let storedComponent = FocusManager._allFocusableComponents[componentId];
-            storedComponent.limitedCount--;
+
+            if (this._isFocusLimited === Types.LimitFocusType.Accessible) {
+                storedComponent.limitedCountAccessible--;
+            } else {
```

Comment:
This code is a bit fragile in that it will break if new elements are added to the LimitFocusType enumeration. I recommend adding an explicit check "else if (this._isFocusLimited === Types.LimitFocusType.Limited)".